drivers/network/dd/dc21x4/dc21x4.h

```diff
@@ -0,0 +1,536 @@
+/*
+ * PROJECT:     ReactOS DC21x4 Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Main header file
+ * COPYRIGHT:   Copyright 2023 Dmitry Borisov <di.sean@protonmail.com>
+ */
+
+#ifndef _DC21X4_PCH_
+#define _DC21X4_PCH_
```

Comment:
you used pragma once in most of your headers. Possible also here? Or is there any specific reasons why classic include-guards are the only option here?

Reply:
Is it allowed to use #pragma once in a precompiled header file?

Reply:
iirc. yes it's possible.

Reply:
Then let's do it.

Reply:
Done.